utils/Cargo.toml

```diff
@@ -14,7 +14,11 @@ edition = "2018"
 bench = false
 
 [features]
+alloc = ["hashbrown"]
 concurrent = ["rayon"]
+default = ["std"]
+std = []
 
 [dependencies]
+hashbrown = { version = "0.9.1", optional = true, default-features = false, features = ["ahash", "inline-more"] }
 rayon = { version = "1.5", optional = true }
```

Comment:
Is there a reason we are not using the latest version of `hashbrown` here? (I believe the latest one is 0.11.2).

Also, nit: could we add a new line at the end of the file? (I know it wasn't there before either).

Reply:
Hmm I'm not sure neither, I thought I had used the latest one. I may have missed up something when moving things here and there. Will correct it, sorry.
As for `alloc` and `concurrent`, I couldn't see which context would require them both neither, but as they are not incompatible per say, I didn't see the need to make them mutually exclusive. What would you prefer?

Reply:
Yep - seems like the latest version of `hashbrown` is 0.11.2 rather than 0.9.1: https://crates.io/crates/hashbrown

Regarding `alloc` and `concurrent`, I think making them mutually exclusive could lead to less potential confusion/errors in the future. So, if not too difficult, would be great to add that to this PR.

Reply:
Alright will do!